### PR TITLE
Add some GC papers

### PR DESCRIPTION
**Pull request by jacob-hughes**

### CODE REVIEW

File: softdev.bib [hunk 1]
```diff
@@ -2698,3 +2698,54 @@ @article{rouvoet20knowing
...
+    year = 2016,
+    month = Jun,
+    journal = "SIGPLAN",
+    volume = 51,
+    number = 6,
+    pages = {570--583}
+}
+
+@article{parkinson2017project,
+    title = "Project snowflake: Non-blocking safe manual memory management in. net",
```
Comment (ltratt) [inline softdev.bib:2731]: This needs to be `Project {Snowflake}: Non-blocking safe manual memory management in {.NET}` (welcome to BibTeX's weird capitalisation rules!).

**Comment by ltratt** [comment]
Instead of "SIGPLAN" can we put the particular conference (or journal or event) the papers were published in?

**Comment by jacob-hughes** [comment]
Sure, I've updated them here: c0e21ee4c617b54c62c6b69400f03cd9141c6465

**Comment by ltratt** [comment]
Please squash.

**Comment by jacob-hughes** [comment]
Pushed a fix for the build failure: 6021f34b103a97fb84634301a7e1d8522675a7b4

**Comment by jacob-hughes** [comment]
Squashed

**Pull request merged**